content/chat/rooms/moderation/hive-model-only.textile

```diff
@@ -9,4 +9,29 @@ languages:
   - kotlin
 ---
 
-Hive documentation to go here.
+"Hive Moderation":https://hivemoderation.com is a powerful suite of moderation tools that can be used to moderate content in chat rooms.
+
+The Hive (Model Only) rule is a rule applied to chat rooms in Ably Chat which allows you to use Hive's text moderation models to detect and handle inappropriate content before it is published to other users.
+
+h2(#fields). Rule fields
+
+- Hive API Key := The API key for your Hive account.
+- Thresholds := A map of text "text moderation classes":https://docs.thehive.ai/reference/text-moderation to "severity":https://docs.thehive.ai/docs/detailed-class-descriptions-text-moderation. When moderating text, any message deemed to be at or above a specified threshold will be rejected and not published to the chat room.
+- Model URL (Optional) := A custom URL if using a custom moderation model.
+- Retry Timeout := Maximum duration (in milliseconds) that an attempt to invoke the rule may take (including any retries).
```

Comment:
Is there a default?

Reply:
No default, must be specified explicitly

Reply:
If no default, is there a range? It's good to provide as much info as needed for the developer to get going without the need to navigate to the hive docs if it can be helped.

Reply:
Yes, added